.github/workflows/ci-bdnbenchmark.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         os: [ ubuntu-latest, windows-latest ]
         framework: [ 'net8.0' ]
         configuration: [ 'Release' ]
-        test: [ 'Operations.BasicOperations', 'Operations.ObjectOperations', 'Cluster.ClusterMigrate', 'Cluster.ClusterOperations', 'Lua.LuaScripts' ]
+        test: [ 'Operations.BasicOperations', 'Operations.ObjectOperations', 'Cluster.ClusterMigrate', 'Cluster.ClusterOperations', 'Lua.LuaScripts', 'Operations.CustomOperations','Operations.RawStringOperations','Operations.ScriptOperations' ]
```

Comment:
There was another merge to main that adds `HashObjectOperations`, so we need to add this to the PR as well. Hopefully this is the last and we can merge after it.

Reply:
Yep ... merged